WooCommerce/Classes/ViewRelated/Dashboard/Blaze/BlazeCampaignDashboardViewModel.swift

```diff
@@ -65,6 +65,18 @@ final class BlazeCampaignDashboardViewModel: ObservableObject {
         stores.sessionManager.defaultSite?.url ?? ""
     }
 
+    var campaignsCount: Int {
+        blazeCampaignResultsController.fetchedObjects.count
```

Comment:
I'm concerned with this count as we get only the number items as we fetched from remote, and we are using pagination so unless we fetched all the pages, the number of items in storage can be incorrect. 

Here's a recording - at first I have 25 items, then after scrolling to another page on the campaign list, I have 38 (final total):


https://github.com/woocommerce/woocommerce-ios/assets/5533851/49a460dd-50cf-4878-a0dd-3b75a25dac8f

So unless we have a way to get the total number of campaigns created (a different endpoint), we should not show the number on the button to avoid edge cases. I'm good with diverging from the design if it's not technically supported.

Reply:
Good point. I took a quick look into the API doc and didn't find a way to just easily get the total number of campaigns (maybe we can request). For now let's go back to just "View all campaigns".

Reply:
Handled in https://github.com/woocommerce/woocommerce-ios/pull/12593/commits/c84c75d2ee204dd45e94b5c24e08e6909d284b5c